src/shared/utils/diagnose.ts

```diff
@@ -80,14 +80,26 @@ export const diagnose = ({
   if (!document) {
     return handleError(debug, "The document must not be empty");
   }
+
   if (typeof document !== "object") {
     return handleError(debug, "The document must be an object");
   }
 
+  if (kind === "raw") {
```

Comment:
this is not good enough. `isRawV2Document` certifies that `document is OpenAttestationDocumentV2` but this check does nothing.

We must perform the requested verification (schema)

Reply:
okay done, change to use getSchema to validate.